runners/katacoda/templates/runServerJava.md

```diff
@@ -0,0 +1,7 @@
+<%= cdCommand; %>
+
+Start the server in terminal <%= terminalId; %> by running the maven command 'mvn spring-boot:run'.
+
+`mvn spring-boot:run`{{execute T<%= terminalId; %> <% if (interrupt) { %>interrupt<% } %>}}
+
+<%= textAfter; %>
```

Comment:
Here we can mention that it may take a few seconds for the server to run. And maybe we can add that the server is stopped and relaunched automatically by executing the command (only if it is running)